src/main/java/org/craftercms/studio/controller/rest/v2/ExceptionHandlers.java

```diff
@@ -49,6 +51,24 @@ public ResponseBody handleAuthenticationException(HttpServletRequest request, Se
         return handleExceptionInternal(request, e, ApiResponse.UNAUTHENTICATED);
     }
 
+    @ExceptionHandler(UserAlreadyExistsException.class)
+    @ResponseStatus(HttpStatus.CONFLICT)
+    public ResponseBody handleUserAlreadyExistsException(HttpServletRequest request, ServiceException e) {
+        ApiResponse response = new ApiResponse(ApiResponse.USER_ALREADY_EXISTS);
+        response.setMessage(response.getMessage() + ": " + e.getMessage());
```

Comment:
No need for this line, the extra details are only necessary for system exceptions.